File-Formats/DocIO/Find-item-in-word-document.md

```diff
@@ -0,0 +1,740 @@
+        WChart chart = document.FindItemByProperties(EntityType.Chart, propertyNames, propertyValues) as WChart;
+        //Rename the ChartTitle.
+        if (chart != null)
+        chart.ChartTitle = "Sales Analysis";
+
+        propertyNames =  new string[] { "Title","Rows.Count" };
+        propertyValues =  new string[]{ "SupplierDetails","6" };
+        //Find table by Title and Rows Count
+        WTable table = document.FindItemByProperties(EntityType.Table, propertyNames,propertyValues) as WTable;
+        //Remove the table in document.
+        if (table != null)
+        table.OwnerTextBody.ChildEntities.Remove(table);
+        //Save a  Word document to the MemoryStream.
+        MemoryStream outputStream = new MemoryStream();
+        document.Save(outputStream, FormatType.Docx);
+        outputStream.Position = 0;
+        //Download as a Word document in the browser.
+        return File(outputStream, "application/msword", "Sample.docx");
+    }
+}
+{% endhighlight %} 
+
+{% highlight c# tabtitle="Xamarin" %}
+//Open the file as Stream.
+using (Stream docStream = typeof(App).GetTypeInfo().Assembly.GetManifestResourceStream("Sample.Assets.Input.docx"))
+{
+    //Load the file stream into a Word document.
+    using (WordDocument document = new WordDocument(docStream, FormatType.Docx))
+    {
+        string[] propertyNames = { "ChartType", "ChartTitle" };
+        string[] propertyValues = { OfficeChartType.Pie.ToString(), "Sales" };
+        //Find chart by ChartType and ChartTitle.
+        WChart chart = document.FindItemByProperties(EntityType.Chart, propertyNames, propertyValues) as WChart;
+        //Rename the ChartTitle.
+        if (chart != null)
+        chart.ChartTitle = "Sales Analysis";
+
+        propertyNames =  new string[] { "Title","Rows.Count" };
+        propertyValues =  new string[]{ "SupplierDetails","6" };
+        //Find table by Title and Rows Count
+        WTable table = document.FindItemByProperties(EntityType.Table, propertyNames,propertyValues) as WTable;
+        //Remove the table in document.
+        if (table != null)
+        table.OwnerTextBody.ChildEntities.Remove(table);
+        //Save a Word document to the MemoryStream.
+        MemoryStream outputStream = new MemoryStream();
+        document.Save(outputStream, FormatType.Docx);
+        //Save the stream as a file in the device and invoke it for viewing. 
+        Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("Sample.docx", "application/msword", outputStream);
+    }
+}
+//Download the helper files from the following link to save the stream as file and open the file for viewing in Xamarin platform
+//https://help.syncfusion.com/file-formats/docio/create-word-document-in-xamarin#helper-files-for-xamarin
+{% endhighlight %} 
+
+{% endtabs %}
+
+## Find all items by property
+
+Using FindAllItemsByProperty API, find all the items in the Word document that has the specified property name and value.
```

Comment:
Using the FindAllItemsByProperty API, find all the items in the Word document with the specified property name and value.